test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'by ID' do
+      expect_sync_plan_search(:name => 'syncplan1', :org_id => 1, :id => 1)
```

Comment:
@chris1984 ```expect_sync_plan_search(1, 'syncplan1', 1)``` (basically replace the ```:name => ...,:org_id => 1, :id => 1, :id => 1) ```   with actual params in all the expect_sync_plan_search calls )